app/controllers/membership_applications_controller.rb

```diff
@@ -21,11 +21,6 @@ def index
   end
 
 
-  def show
-    @categories = @membership_application.business_categories
-  end
```

Comment:
Might it be better to have an empty action rather than no action at all?  (that way it is clear that we actually _do_ have a `show` action, but that its only requirement is to fetch the app and it's categories (which is being done by the `before_action` method)).

Also, we are calling the `get_membership_application` before the `destroy`, but we really only need to fetch the membership_application and _not_ its categories.  Perhaps refactor?